automation/jenkins/aws/manageDocker.sh

```diff
@@ -248,6 +260,7 @@ DOCKER_TAG="${DOCKER_TAG:-${DOCKER_TAG_DEFAULT}}"
 DOCKER_IMAGE_SOURCE="${DOCKER_IMAGE_SOURCE:-${DOCKER_IMAGE_SOURCE_DEFAULT}}"
 DOCKER_OPERATION="${DOCKER_OPERATION:-${DOCKER_OPERATION_DEFAULT}}"
 DOCKER_PRODUCT="${DOCKER_PRODUCT:-${PRODUCT}}"
+DOCKER_CONTEXT_DIR="${DOCKER_CONTEXT_DIR:-${DOCKER_CONTEXT_DIR_DEFAULT}}"
```

Comment:
Looks like the semantics have changed a little here? So if existing configs explicitly set `DOCKER_CONTEXT_DIR`, they will need to be changed?

Reply:
Yeah there would be some minor updates. However since setting the context dir runs from the default directory I think it ends up being the same place that was used before anyway. I.e. the CONTEXT_DIR used to base itself on the workspace directory. So you would be in the same place by default